src/main/java/frc/robot/commands/intake/IntakeFlip.java

```diff
@@ -1,21 +1,20 @@
 /* ==================================================
-Authour: Shane Pinto
-Description: IntakeDeploy.java is a command that 
-tells the intake subsystem to deploy the intake 
-piston.
+Author: Lucas Jacobs
+Description: IntakeFlip.java is a command that deploys the intake
+if it is retracted and retracts the intake if it is deployed
 ================================================== */
 
 package frc.robot.commands.intake;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.Robot;
 
-public class IntakeDeploy extends CommandBase
+public class IntakeFlip extends CommandBase
 {
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
   private Robot robot;
 
-  public IntakeDeploy(Robot robot)
+  public IntakeFlip(Robot robot)
```

Comment:
We should still keep the `IntakeDeploy` and `IntakeRetract` commands as well as `IntakeFlip`.  This will be useful in autonomous, where you may only want to deploy or retract the intake at certain points in time.  There won't be anyone using the controllers to toggle the intake.

Reply:
Good idea.